master/static/migrations/20210211000000_add-task-session.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE SEQUENCE public.task_sessions_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+CREATE TABLE public.task_sessions (
```

Comment:
Not super important, but we may want to have some way to clean up old sessions in case anything gets left around after a hard crash of the master.

Reply:
We did. There is a function to clean up all the task sessions when the master restarts.